lib/gems/pending/util/MiqSshUtil.rb

```diff
@@ -38,6 +69,9 @@ def initialize(host, user, password, options = {})
     end
   end # def initialize
 
+  # Download the contents of the remote +from+ file to the local +to+ file. Some
+  # messages will be written to the global ManageIQ log in debug mode.
+  #
   def get_file(from, to)
     run_session do |ssh|
       $log&.debug("MiqSshUtil::get_file - Copying file #{@host}:#{from} to #{to}.")
```

Comment:
I realize this is a documentation PR, but it would be nice to update the calls to `$log.debug` to be consistent.  Some use this safe-navigation approach but many still use a `if $log` check.

In additional, the calls can use a block so the string is not interpolated unless the logging level is active:
`$log&.debug { "MiqSshUtil::get_file - Copying file #{@host}:#{from} to #{to}." }`

Just some thoughts for a followup PR.

Reply:
Yeah, there's some oddities with the logging, and I'm not entirely sure why the log isn't set on the ssh object itself. That part was commented out for some reason, though I don't know why.